### PR TITLE
MultiCheckbox component improvements

### PR DESCRIPTION
**Pull request by magicznyleszek**
## Description

Make the frame around MultiCheckbox component optional. Add the component to design system.

## Related issues

Needed for #4051

### CODE REVIEW

File: jsapp/js/components/common/multiCheckbox.tsx [hunk 1]
```diff
@@ -1,23 +1,33 @@
...
+  /** Use this to disable all checkboxes - useful for blocking changes while loading. */
+  disabled?: boolean;
   /** Returns whole list whenever any item changes */
   onChange: (items: MultiCheckboxItem[]) => void;
 }
 
-/** A MultiCheckbox generic component. */
+/**
+ * A MultiCheckbox generic component.
+ * Use optional `bem.MultiCheckbox__wrapper` to display a frame around it.
```
Comment (duvld) [inline jsapp/js/components/common/multiCheckbox.tsx:29]: I wonder why not put the wrapper around the component, and have an optional prop enable the wrapper in the component itself?

File: jsapp/js/designSystem/multiCheckboxDemo.tsx [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+                        onChange={this.onIsDisabledChange.bind(this)}
+                        checked={this.state.demoIsDisabled}
+                      />
+                    </div>
+                  </div>
+                </form>
+              </bem.SimpleTable__cell>
+              <bem.SimpleTable__cell>
+                <div className='demo__preview'>
+                  <MultiCheckbox
```
Comment (duvld) [inline jsapp/js/designSystem/multiCheckboxDemo.tsx:157]: Should we demo the wrapper as well?

File: jsapp/js/components/common/checkbox-and-radio.scss [hunk 3]
```diff
@@ -28,7 +29,7 @@
   }
 
   .radio__row {
-    display: block;
+    padding: 4px 0;
```
[the remark this answers is not in the document]
Comment (magicznyleszek) [reply jsapp/js/components/common/checkbox-and-radio.scss]: Right! Fixed :)

File: jsapp/js/components/common/multiCheckbox.tsx (hunk 1 above)
Comment (magicznyleszek) [reply jsapp/js/components/common/multiCheckbox.tsx:31]: Good idea, changed it by adding a `type` prop :)

File: jsapp/js/designSystem/multiCheckboxDemo.tsx (hunk 2 above)
Comment (magicznyleszek) [reply jsapp/js/designSystem/multiCheckboxDemo.tsx:184]: Yes, added :)

**Pull request merged**